values-schema.yaml

```diff
@@ -2316,40 +2353,43 @@ properties:
               - duration
           storage:
             properties:
-              aws:
-                type: object
               azure:
-                description: Missing from docs but available with this schema.
                 properties:
-                  account_key:
-                    type: string
+                  accountName:
+                    $ref: '#/definitions/wordCharacterPattern'
+                  containerName:
+                    $ref: '#/definitions/wordCharacterPattern'
+                  accountKey:
+                    $ref: '#/definitions/wordCharacterPattern'
+                    x-secret: ''
+              s3:
+                properties:
+                  bucket:
+                    $ref: '#/definitions/wordCharacterPattern'
+                  s3Url:
+                    $ref: '#/definitions/wordCharacterPattern'
+                  accessKeyId:
+                    $ref: '#/definitions/wordCharacterPattern'
+                  secretAccessKey:
+                    $ref: '#/definitions/wordCharacterPattern'
                     x-secret: ''
-                  account_name:
-                    type: string
-                  container_name:
-                    type: string
-                type: object
-              bigtable:
-                type: object
-              cassandra:
-                type: object
               gcs:
-                type: object
-              swift:
-                type: object
-              local:
-                type: object
-              storageType:
```

Comment:
Here things go wrong: schema has been changed but there are no migrations defined in the `values-changes.yaml` file